lang/java/compiler/src/main/velocity/org/apache/avro/compiler/specific/templates/java/classic/record.vm

```diff
@@ -28,6 +28,9 @@ import org.apache.avro.message.BinaryMessageDecoder;
 import org.apache.avro.message.SchemaStore;
 #end
 #if (${this.gettersReturnOptional} || ${this.createOptionalGetters})import java.util.Optional;#end
+#if (${this.createNullSafeAnnotations})import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.NotNull;
+#end
```

Comment:
Instead of an import, can we please use the same pattern as for other types?

Instead of `String`, the code used `java.lang.String` as well, because this makes name clashes less likely.

Additionally, using explicit annotation classes allows us to make the annotations configurable.